apps/traincascade/cascadeclassifier.cpp

```diff
@@ -450,6 +452,7 @@ void CvCascadeClassifier::save( const string filename, bool baseFormat )
         CvSeq* weak;
         if ( cascadeParams.featureType != CvFeatureParams::HAAR )
             CV_Error( CV_StsBadFunc, "old file format is used for Haar-like features only");
+        fs << "{:" << ICV_HAAR_TYPE_ID;
```

Comment:
why "{:" is used here instead of "{"? In the case of YAML format it will result in very differently-looking representation